test/integration/routing.go

```diff
@@ -174,3 +193,48 @@ func (t *routing) verifyFaultInjection(src, dst, headerKey, headerVal string,
 	}
 	return nil
 }
+
+// verifyRedirect verifies if the http redirect was setup properly
+func (t *routing) verifyRedirect(src, dst, targetHost, targetPath, headerKey, headerVal string, respCode int) error {
+	url := fmt.Sprintf("http://%s/%s", dst, src)
+	glog.Infof("Making 1 request (%s) from %s...\n", url, src)
+
+	resp := t.clientRequest(src, url, 1, fmt.Sprintf("-key %s -val %s", headerKey, headerVal))
+
+	statusCode := ""
```

Comment:
Don't need to convert the code to an integer:
`if len(resp.code) == 0 || resp.code[0] != fmt.Sprint(respCode) { ...return error... }`